tensorflow-mixin/dashboards/tensorflow-overview.libsonnet

```
@@ -718,23 +732,23 @@ local containerLogsPanel = {
         std.flattenArrays([
           // Model Row
           [
-            modelRequestRatePanel { gridPos: { h: 8, w: 24, x: 0, y: 0 } },
-            modelPredictRequestLatencyPanel { gridPos: { h: 8, w: 12, x: 0, y: 8 } },
-            modelPredictRuntimeLatencyPanel { gridPos: { h: 8, w: 12, x: 12, y: 8 } },
+            modelRequestRatePanel(getMatcher($._config)) { gridPos: { h: 8, w: 24, x: 0, y: 0 } },
+            modelPredictRequestLatencyPanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 0, y: 8 } },
+            modelPredictRuntimeLatencyPanel(getMatcher($._config)) { gridPos: { h: 8, w: 12, x: 12, y: 8 } },
           ],
           // Serving Overview Row
           [
-            servingOverviewRow { gridPos: { h: 1, w: 24, x: 0, y: 16 } },
-            graphBuildCallsPanel { gridPos: { h: 8, w: 12, x: 0, y: 17 } },
-            graphRunsPanel { gridPos: { h: 8, w: 12, x: 12, y: 17 } },
-            graphBuildTimePanel { gridPos: { h: 8, w: 12, x: 0, y: 25 } },
-            graphRunTimePanel { gridPos: { h: 8, w: 12, x: 12, y: 25 } },
-            batchQueuingLatencyPanel { gridPos: { h: 8, w: 12, x: 0, y: 33 } },
-            batchQueueThroughputPanel { gridPos: { h: 8, w: 12, x: 12, y: 33 } },
+            servingOverviewRow(getMatcher($._config)) { gridPos: { h: 1, w: 24, x: 0, y: 16 } },
```

Row does not need the matcher
```suggestion
            servingOverviewRow { gridPos: { h: 1, w: 24, x: 0, y: 16 } },
```